src/mailadm/commands.py

```diff
@@ -96,6 +97,21 @@ def list_tokens(db) -> str:
     return '\n'.join(output)
 
 
+def qr_from_token(db, tokenname):
+    with db.read_connection() as conn:
+        token_info = conn.get_tokeninfo_by_name(tokenname)
+        config = conn.config
+
+    if token_info is None:
+        return {"status": "error",
+                "message": "token {!r} does not exist".format(tokenname)}
+
+    image = gen_qr(config, token_info)
```

Comment:
Unfortunately, this still raises an error:

```
2022-08-16T12:04:29.097343606Z Exception /usr/lib/python3.9/site-packages/mailadm/data/opensans-regular.ttf
2022-08-16T12:04:29.097350171Z Traceback:
2022-08-16T12:04:29.097356317Z Traceback (most recent call last):
2022-08-16T12:04:29.097362463Z   File "/usr/lib/python3.9/site-packages/deltachat/events.py", line 261, in swallow_and_log_exception
2022-08-16T12:04:29.097368469Z     yield
2022-08-16T12:04:29.097373917Z   File "/usr/lib/python3.9/site-packages/deltachat/events.py", line 256, in _inner_run
2022-08-16T12:04:29.097379923Z     hook(**kwargs)
2022-08-16T12:04:29.097385790Z   File "/usr/lib/python3.9/site-packages/pluggy/_hooks.py", line 265, in __call__
2022-08-16T12:04:29.097404507Z     return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
2022-08-16T12:04:29.097410514Z   File "/usr/lib/python3.9/site-packages/pluggy/_manager.py", line 80, in _hookexec
2022-08-16T12:04:29.097416311Z     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
2022-08-16T12:04:29.097422596Z   File "/usr/lib/python3.9/site-packages/pluggy/_callers.py", line 60, in _multicall
2022-08-16T12:04:29.097429022Z     return outcome.get_result()
2022-08-16T12:04:29.097434888Z   File "/usr/lib/python3.9/site-packages/pluggy/_result.py", line 60, in get_result
2022-08-16T12:04:29.097441244Z     raise ex[1].with_traceback(ex[2])
2022-08-16T12:04:29.097447180Z   File "/usr/lib/python3.9/site-packages/pluggy/_callers.py", line 39, in _multicall
2022-08-16T12:04:29.097465548Z     res = hook_impl.function(*args)
2022-08-16T12:04:29.097471694Z   File "/usr/lib/python3.9/site-packages/mailadm/bot.py", line 61, in ac_incoming_message
2022-08-16T12:04:29.097477002Z     fn = qr_from_token(self.db, arguments[1])["filename"]
2022-08-16T12:04:29.097495510Z   File "/usr/lib/python3.9/site-packages/mailadm/commands.py", line 109, in qr_from_token
2022-08-16T12:04:29.097502564Z     image = gen_qr(config, token_info)
2022-08-16T12:04:29.097508850Z   File "/usr/lib/python3.9/site-packages/mailadm/gen_qr.py", line 34, in gen_qr
2022-08-16T12:04:29.097527078Z     assert os.path.exists(ttf_path), ttf_path
2022-08-16T12:04:29.097533434Z AssertionError: /usr/lib/python3.9/site-packages/mailadm/data/opensans-regular.ttf
```

the font doesn't seem to be installed in the docker container :/

Reply:
fixed it, a setup.cfg option was missing, no idea why it literally worked everywhere else.